server.py

```diff
@@ -0,0 +1,23 @@
+import socket
+
+
+def main():
+    serv = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    serv.bind(('localhost', 50000))
+    serv.listen(5)
+    conn, addr = serv.accept()
+    from_client = ''
+    data = "Start"
+    while data != "":
```

Comment:
You're already performing this check in line 13, the `data = "Start"` line and this condition are redundant, change this to a `while True`